### PR TITLE
docs: rename Microsoft to microsoft

### PR DESCRIPTION
**Pull request by markwolff**
rename travis and github links to solve issues from org rename to microsoft.

### CODE REVIEW

File: API-reference.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ Parameter | Description
 `handledAt` | Deprecated. This argument is ignored. Please pass `null`.
 `properties` | Map of string to string: Additional data used to [filter exceptions](https://azure.microsoft.com/documentation/articles/app-insights-api-custom-events-metrics/#properties) in the portal. Defaults to empty.
 `measurements` | Map of string to number: Metrics associated with this page, displayed in Metrics Explorer on the portal. Defaults to empty.
-`severityLevel` | Supported values: [SeverityLevel.ts](https://github.com/Microsoft/ApplicationInsights-JS/blob/master/JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/SeverityLevel.ts)
+`severityLevel` | Supported values: [SeverityLevel.ts](https://github.com/microsoft/ApplicationInsights-JS/blob/master/JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/SeverityLevel.ts)
```
Comment (hectorhdzg) [inline API-reference.md]: Can you include more information about this change?

Comment (markwolff) [reply API-reference.md]: The github organization was renamed from "Microsoft" to "microsoft", so this change is just a find&replace on this

Comment (hectorhdzg) [reply API-reference.md]: I got the actual change, my question is more related to why changing the org name if is going to break tons of people, how did you figure out the name was updated?

Comment (markwolff) [reply API-reference.md]: Other repos had similar PRs (e.g. https://github.com/microsoft/ApplicationInsights-Go/pull/30) that showed up in notifcations. Unsure why the org change happened though.

**Pull request merged**